routes/message.routes.js

```diff
@@ -5,16 +5,39 @@ router.get("/message", (req, res) => {
   res.json("Hello world");
 });
 
-
 router.post("/messages", (req, res) => {
   const { destination, message } = req.body;
 
-  const body = message
+  const body = message;
 
   MessageAppService.sendMessage(destination, body)
     .then((response) => res.json(response.data))
-    .catch((err) => res.json(err));
+    .catch((err) => {
+      if (
+        !err.config.data.includes("destination") &&
+        !err.config.data.includes("body")
+      ) {
+        return res.status(405).json({
```

Comment:
Can you explain why you choose these status codes?

Reply:
The status code between 400 and 499 are client errors, so I decided to start the status after 404, which is page no found, so I know which error is happening depending on the status error, but pointing that is a client error.

Reply:
That was my logic yesterday, before knowing that there is specific errors for each status code.